include/APM/APMDevice.hpp

```diff
@@ -32,7 +33,7 @@ class APMDevice {
      * Initializes the IO Devices
      * @param baud the baudrate for the UART device
      */
-    explicit APMDevice(IO::GPIO &accessorySwGpio, APMUart &apmUart, IO::GPIO &chargeSwGpio,
```

Comment:
NIT: Didn't catch this before, but I think the name APMDevice may blur the lines between what a device is and what a board is. Devices are really the components that are on a board. So I think maybe this would just be called APM, or maybe APMPlatform?

Reply:
Yea good point.  The class is supposed to be the top level object holding reference to other APM objects and contains the state transition logic.  I don't like APM, so I think APMPlatform will do.